oxrdflib/parser.py

```diff
@@ -47,6 +47,7 @@ def parse(
 
         else:
             base_iri = sink.absolutize(source.getPublicId() or source.getSystemId() or "")
+            graph_identifier = to_ox(sink.identifier) if format not in ("ox-nq", "ox-nquads") else None
```

Comment:
We should map the default graph for dataset formats too.

```suggestion
            graph_identifier = to_ox(sink.identifier)
```